kong/plugins/apifortress/md5.lua

```diff
@@ -0,0 +1,383 @@
+local md5 = {
+  _VERSION     = "md5.lua 1.0.2",
+  _DESCRIPTION = "MD5 computation in Lua (5.1-3, LuaJIT)",
+  _URL         = "https://github.com/kikito/md5.lua",
+  _LICENSE     = [[
+    MIT LICENSE
```

Comment:
same as with the Base64; please use these instead; https://github.com/openresty/lua-nginx-module#ngxmd5